instance/static/html/instance/appserver.html

```diff
@@ -75,12 +75,12 @@ <h4>{{ appserver.name }}</h4>
           <td>{{ appserver.email }}</td>
         </tr>
         <tr>
-          <td>GitHub admin organizations</td>
-          <td>{{ appserver.github_admin_organizations }}</td>
+          <td>Owner organization</td>
+          <td>{{ instance.owner_organization }}</td>
         </tr>
         <tr>
-          <td>GitHub admin users</td>
-          <td>{{ appserver.github_admin_users }}</td>
+          <td>configuration_source_repo_url</td>
```

Comment:
This seems a typo, we don't want `configuration_source_repo_url` here. I think we wanted to show the github admin users. Is this even possible? Do we store them? Note that this (if we have it) should show the *appserver* admins, not the instance admins (same as the other details, it relates to the appserver).
If we don't *store* the appserver's admins, then I think that we could add the admin users to the instance view (instance admins). But this gets more complex and we don't have time to test this. So I'd prefer not to show this field.

Reply:
@taranjeet can you take a look into this?

Reply:
@clemente @ahmedaljazzar @taranjeet Oops, my bad -- I had suggested in [this comment](https://github.com/open-craft/opencraft/pull/357#discussion_r223975991) to show the creator of the app server here (which should really be the creator of the parent instance), but ended up not updating the code snippet appropriately before posting the comment. Here's an updated version of that snippet:

```html
        <tr>
          <td>Owner organization</td>
          <td>{{ instance.owner_organization }}</td>
        </tr>
        <tr>
          <td>Instance creator (Ocim user)</td>
          <td>{{ instance.creator_username }}</td>
        </tr>
```

I agree that we shouldn't look into displaying the list of users that have SSH access to the app server here. Simply updating the diff to match the snippet above will be enough.